corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -179,7 +188,7 @@ def consumption_entry(case_id, product_id, section_id):
 
 
 def get_case_payload(domain, stock_settings, version, user, last_sync, synclog):
-        response_elements = []
+        response = StringRestoreResponse()
```

Comment:
This is where the real improvement is. Passing a StringRestoreResponse object back rather than a list of XML element classes.